recipes/metamorpheus/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -eox pipefail
+
+PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
+
+mkdir -p $PREFIX/bin
+cp -r $SRC_DIR/* $PREFIX
+
+if [[ "${build_platform}" == "win-64" ]]; then
+    mv $PREFIX/CMD.exe $PREFIX/metamorpheus.exe
+else
+    cp $RECIPE_DIR/metamorpheus $PREFIX/metamorpheus
```

Comment:
Normally binaries are installed into `$PREFIX/bin` rather than directly into `$PREFIX`

Reply:
It looks like all the `metamorpheus` files are being installed into `$PREFIX`!

![image](https://user-images.githubusercontent.com/881019/113706821-760a4980-9722-11eb-801d-66a36b4bc2ab.png)

* https://artprodeus21.artifacts.visualstudio.com/A910fa339-c7c2-46e8-a579-7ea247548706/84710dde-1620-425b-80d0-4cf5baca359d/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2NvbmRhLWZvcmdlL3Byb2plY3RJZC84NDcxMGRkZS0xNjIwLTQyNWItODBkMC00Y2Y1YmFjYTM1OWQvYnVpbGRJZC8zMDAxMTIvYXJ0aWZhY3ROYW1lL2NvbmRhX3BrZ3Nfd2lu0/content?format=zip

Reply:
I think `metamorpheus` should probably be installed into it's own folder - maybe `$PREFIX/dotnet/tools/metamorpheus`? That's the location I installed [`dotnet-interactive`](https://github.com/conda-forge/dotnet-interactive-feedstock/blob/master/recipe/bld.bat)

To make the executable available on the path you could then add an executable shim in `$PREFIX/bin` or maybe `$PREFIX/dotnet/tools`
e.g. https://github.com/conda-forge/dotnet-feedstock/blob/master/recipe/dotnet.cmd